src/Hazelcast.Net/HazelcastOptions.cs

```diff
@@ -100,6 +96,13 @@ private HazelcastOptions(HazelcastOptions other)
         [BinderIgnore(false)]
         internal CoreOptions Core { get; } = new CoreOptions();
 
+        /// <summary>
+        /// Gets the <see cref="HazelcastFailoverOptions"/>.
+        /// </summary>
+        /// <returns>The failover options.</returns>
+        [BinderIgnore(false)]
+        internal HazelcastFailoverOptions FailoverOptions { get; set; } = new HazelcastFailoverOptions();
```

Comment:
Is this visible to the user in any way? Also, why do we carry the failover options in the client options?

Reply:
It's internal. The purpose is passing the failover options without any refactoring since `HazelcastOptions` were already passed to everywhere. Also, if we don't carry the `FailoverOptions` with in `HazelcastOptions` we may need a type check of `options` in everywhere to act accordingly.